app/assets/src/components/views/SampleView/SampleView.jsx

```diff
@@ -561,12 +561,25 @@ class SampleView extends React.Component {
     } = this.props;
 
     const showAMR = amr;
+    const pipelineVersion =
+      pipelineRun && pipelineRun.version && pipelineRun.version.pipeline;
     return (
       <div>
         <NarrowContainer>
           <ViewHeader className={cs.viewHeader}>
             <ViewHeader.Content>
-              <div className={cs.pipelineInfo}>
+              <div
+                className={cx(
+                  cs.pipelineInfo,
+                  pipelineVersion && cs.linkToPipelineViz
+                )}
+                onClick={() =>
+                  pipelineVersion &&
+                  window.open(
```

Comment:
Why not use an anchor tag on the version display below?
Does this interfere with the select box?

Reply:
Switched to an anchor tag — I think I just forgot that they existed yesterday 😅

Reply:
Actually, I switched back (I forgot to start npm when developing). The anchor tag interferes with the select box.